exonum_launcher/launcher.py

```diff
@@ -123,19 +123,27 @@ def deploy_all(self) -> None:
 
     def wait_for_deploy(self) -> None:
         """Waits for all the deployments to be completed."""
-        pending_deployments = self.launch_state.pending_deployments()
-        for tx_hashes in pending_deployments.values():
-            self._explorer.wait_for_txs(tx_hashes)
-
-        for artifact in pending_deployments:
-            result = self._explorer.wait_for_deploy(artifact)
-            self.launch_state.complete_deploy(artifact, result)
-
-    def start_all(self) -> None:
+        descriptions: Dict[Artifact, str] = dict()
+        check_for_deploy: List[Artifact] = list()
+        for artifact, tx_hashes in self.launch_state.pending_deployments().items():
+            try:
+                self._explorer.wait_for_txs(tx_hashes)
+                descriptions[artifact] = "deployed is ok"
```

Comment:
Nit: "deployed is ok" looks ungrammatic; maybe, use something like "deployed successfully"?